evm_loader/lib/src/types/tracer_ch_db.rs

```diff
@@ -363,6 +366,29 @@ impl ClickHouseDb {
         Ok(account)
     }
 
+    pub async fn get_accounts_in_transaction(&self, sol_sig: &[u8]) -> ChResult<Vec<AccountData>> {
+        info!("get_accounts_in_transaction {{signature: {sol_sig:?} }}");
+
+        let query = r"
+            SELECT pubkey, owner, lamports, data, executable, rent_epoch
+            FROM events.update_account_distributed
+            WHERE txn_signature = ?";
```

Comment:
The same transaction can be inserted into different slots (on different branches).
    So, several transactions with the same sol_sig exist in the database. You have to
    distinguish it by slot number.